Bash_scripts/HW28/Jenkinsfile_1_2_3

```diff
@@ -0,0 +1,31 @@
+pipeline {
+  agent any
+  stages {
+    stage('Shell script 0') {
+      steps {
+        sh '''
+        #!/bin/bash
+        cd $PATHTO
+        SHELL_SCRIPTS=$(find . -name \'*.sh\')
+        for var in $SHELL_SCRIPTS ; do
+        shellcheck "$var"
+        done
+        '''
+      }
+    }    
+    stage('Shell script 1') {
+      steps {
+        sh '''
+        #!/bin/bash
+        DOCKERFILE=$(find "$PATHTO" -name \'Dockerfile\')
```

Comment:
Linting should be done only for Dockerfile(s) you are going to use, otherwise your pipeline can be block with not relevant Dockerfile problem.

Reply:
fixed